api/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN pip install --upgrade pip && pip install -r requirements.txt
 COPY . $API_HOME  
 
 # make migrations
-RUN python manage.py makemigrations && python manage.py migrate
+RUN sh scripts/makemigrations.sh && python manage.py migrate
```

Comment:
`makemigrations.sh` should be executed independently by `./scripts/makemigrations`

That's how we are expecting it to be run in local and deployed env. The behaviour should be consistent.